README.md

```diff
@@ -5,9 +5,11 @@ HTTP wrapper for [Bloomberg Open API].
 
 Find source code in the [GitHub repository].
 
+For information on how to use the HTTP wrapper, see the [Client Guide].
```

Comment:
Yeah, wasn't sure what to name this -- I was trying to use different terminology to distinguish between the HTTP API and the underlying Bloomberg Open API. API Protocol Guide is weird because it could be confused with the latter, but I am not happy with "Client Guide" either.